web/ui/react-app/src/pages/Config.tsx

```diff
@@ -1,6 +1,46 @@
-import React, { FC } from 'react';
+import React, { FC, useEffect, useState } from 'react';
 import { RouteComponentProps } from '@reach/router';
+import { Alert, Button } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+import CopyToClipboard from 'react-copy-to-clipboard';
 
-const Config: FC<RouteComponentProps> = () => <div>Config page</div>
+import './Config.css';
+
+const Config: FC<RouteComponentProps> = () => {
+  const [config, setConfig] = useState(null);
+  const [error, setError] = useState("");
+  const [copied, setCopied] = useState(false);
+
+  useEffect(() => {
+    fetch('../api/v1/status/config')
+      .then(res => res.json())
+      .then(res => setConfig(res.data.yaml))
+      .catch(error => setError(error.message));
+  });
+
+  return (
+    <>
+      <h2>
+        Configuration&nbsp;
+        <CopyToClipboard
+          text={config ? config! : ''}
+          onCopy={() => { setCopied(true); setTimeout(() => setCopied(false), 1500);}}
```

Comment:
Isn't better to pass "result" param in case copy is not successful? ```(text, result) => setCopied(result)...```
Also, you can omit the extra callback in the timeout ```setTimeout(setCopied, 1500)``` should be enough

Reply:
Ah, I just copied the example from https://www.npmjs.com/package/react-copy-to-clipboard, didn't realize it had a `result` param, thanks! And yeah :)